### ISSUE TITLE
Create a main class

### ISSUE THREAD

**Issue by atoomer** [MineGamer69/ParkingApp#1]
Eventually we will need to create a class that lets us run/ use all of our classes together in an organizes way. Then we can tweak out existing code to match it. (I wrote this issue for assignment 9).

**Comment by AaryanKapoor22** [comment]
Yeah I agree with you since we should have one class that controls all other classes and make it easily accessable for the developers to link their code and make it all a part of the main class, I agree it will help us organize much better for our app!

**Comment by AJArmy60** [comment]
This would be a good way for us to test whether or not all of our methods are compatable/can actually work together.  Afterwards we can change names/functionalities if necessary to make sure things line up.
